pkg/scheduler/plugins/defaults.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+import "github.com/kubernetes-sigs/kube-batch/pkg/scheduler/conf"
+
+// ApplyPluginConfDefaults sets option's filed to its default value if not set
+func ApplyPluginConfDefaults(option *conf.PluginOption) {
+	t := true
+
+	switch option.Name {
+	case "conformance":
```

Comment:
It's not necessary to check plugins, just set `true` as default value :)

Reply:
Finally get it now! :)